### PR TITLE
Updated outcomes types content

### PR DESCRIPTION
**Pull request by ralph-hawkins**
* simplified some of the types
* replaced all the '/' with 'and'
* removed the things in brackets (this content will be added to the supporting information)

### CODE REVIEW

**Comment by quis** [comment]
:+1: @allait Does this mean regenerating the schemas?

File: frameworks/digital-outcomes-and-specialists/questions/services/supportAndOperationsTypes.yml [hunk 1]
```diff
@@ -12,5 +12,5 @@ options:
   - label: Monitoring
   - label: Network administration
   - label: Systems administration
-  - label: Technical service desk
-  - label: Tooling (continuous integration / deployment tools)
+  - label: IT service desk
```
Comment (TheDoubleK) [inline frameworks/digital-outcomes-and-specialists/questions/services/supportAndOperationsTypes.yml]: Is "IT" really better than "Technical"? Could it just be "Service desk"?

Comment (ralph-hawkins) [inline frameworks/digital-outcomes-and-specialists/questions/services/supportAndOperationsTypes.yml]: Agreed. Service Desk is better.

File: frameworks/digital-outcomes-and-specialists/questions/services/securityTypes.yml [hunk 2]
```diff
@@ -5,11 +5,12 @@ depends:
...
+  - label: CESG information assurance certification
+  - label: Firewall audit
+  - label: Incident response and forensics
   - label: Infrastructure review
   - label: Risk management
   - label: Security policy
-  - label: THC (Pen) Testing
+  - label: IT health check
   - label: Threat modelling
+  - label: Vulnerability and penetration testing
```
Comment (robyoung) [inline frameworks/digital-outcomes-and-specialists/questions/services/securityTypes.yml]: This isn't a unix text file. It needs a newline at the end of the file.

**Comment by TheDoubleK** [comment]
I think it will mean regenerating schemas, as allowed values have changed.

**Comment by robyoung** [comment]
I have rebased on the new, updated master.

**Pull request merged**